crates/bevy_reflect/src/impls/std.rs

```diff
@@ -591,6 +594,19 @@ mod tests {
         let _serializable = reflect_serialize.get_serializable(&std::time::Duration::ZERO);
     }
 
+    #[test]
+    fn can_serialize_phantomdata() {
```

Comment:
Can we add a test for deserializing a struct that contains `PhantomData` as well?